code/game/objects/items/storage/briefcase.dm

```diff
@@ -35,6 +35,24 @@
 	new /obj/item/stamp/law(src)
 	..()
 
+/obj/item/storage/briefcase/suicide_act(mob/user)
+	var/list/papers_found = list()
+	var/turf/T = get_turf(src)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-descriptive-and-obvious-names